fallout/mob/falloutmobs.dm

```diff
@@ -0,0 +1,665 @@
+
+/mob/living/simple_animal/fallout
+	name = "fallout mob"
+	desc = "If you see this, kill it, purge it and call the developer hotline, we will beat those mappers/admins, lickety split!"
+	icon = 'fallout/icons/mob/falloutanimals.dmi'
+	icon_state = ""
+	gender = NEUTER
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	speak = list()
+	speak_emote = list()
+	emote_hear = list()
+	emote_see = list()
+	speak_chance = 15
+	turns_per_move = 5
+	see_in_dark = 6
+	butcher_results = list()
+	guaranteed_butcher_results = list()
+	loot = list()
+	response_help_continuous = "pats"
+	response_help_simple = "pat"
+	response_disarm_continuous = "pushes aside"
+	response_disarm_simple = "push aside"
+	response_harm_continuous = "kicks"
+	response_harm_simple = "kick"
+	attack_verb_continuous = "kicks"
+	attack_verb_simple = "kick"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	speak_emote = list()
+	health = 100
+	maxHealth = 100
+	speed = 2
+	blood_volume = BLOOD_VOLUME_NORMAL
+	food_type = list()
+	tame_chance = 25
+	bonus_tame_chance = 15
+	footstep_type = FOOTSTEP_MOB_SHOE
+	stop_automated_movement_when_pulled = 1
+	//does the homie have chemss/chems to extract? Poison, Milk, Other :flushed:
+	var/milkable = FALSE
+	//Chems that the creature makes
+	var/extract = null
+	var/obj/item/fallout/animalchem/chems = null
+	//does the homie lay eggs?
+	var/eggable = FALSE
+	var/egg_type = null
+	var/eggsleft = 0
+	//can the homie be ridden?
+	var/rideable = FALSE
+	//does the homie have a saddle to ride?
+	var/saddled = FALSE
+	//feed to clone system working?
+	var/breedable = FALSE
+	//small sprite lad? Jostle em
+	var/smallasslad = FALSE
+	//32x64 sprite lad?
+	var/largeasslad = FALSE
+	//is it a bambino doe?
+	var/is_young = FALSE
+	//do they got a bambino tho????
+	var/has_young = FALSE
+	//bambino
+	var/young_type = null
+	//adult
+	var/adult_type = null
+	//hunger for tamed mobs, dictates starvation and things like production of eggs/milk and such
+	var/hunger = 200
+	var/maxhunger = 200
+	//changes the players sprite adjustment when riding
+	var/offsetx = 0
+	var/offsety = 0
+
+/mob/living/simple_animal/fallout/Initialize()
+	icon_dead = "[icon_state]_dead"
+	var/matrix/bambinoscale = matrix()
+	if(is_young == TRUE)
+		bambinoscale.Scale(0.7, 0.7)
+		transform = bambinoscale
+		return
+	if(smallasslad == TRUE)
+		//makes them small ass lads look all funky n shi
+		pixel_x = rand(-6, 6)
+		pixel_y = rand(0, 10)
+	if(largeasslad == TRUE)
+		pixel_x = -16
+	if(tame)
+		if(milkable == TRUE)
+			chems = new()
+			return . = ..()
+		if(eggable == TRUE)
+			eggsleft = 0
+	else
+		return . = ..()
+
+/mob/living/simple_animal/fallout/Destroy()
+	if(milkable == TRUE)
+		qdel(chems)
+		chems = null
+		return ..()
+	return . = ..()
+
+//hunger and baby grow/birth/speed shitcode
+/mob/living/simple_animal/hostile/fallout/Life()
+	. =..()
+	if(tame)
+		var/hungerrate = rand(0.2,0.3)
+		hunger -= hungerrate
+		return
+		if(hunger == 0)
+			hungerrate = 0
+			return
```

Comment:
Why are you returning here? and why are you returning before the if statement? it's dead code